### PR TITLE
Nits for PHP 8 announcement page

### PR DESCRIPTION
**Pull request by Girgias**
Translations updated by abusing Google Translated and my limited knowledge of said languages

### CODE REVIEW

File: releases/8.0/de.php [hunk 1]
```diff
@@ -424,11 +424,17 @@ public function __construct(
         </li>
         <li>
           Typen für interne Funktionen
-          <a href="https://externals.io/message/106522">RFC</a>
+          <a href="https://externals.io/message/106522">E-Mail-Thread</a>
         </li>
         <li>
-          Curl verwendet nun Objekte anstelle des resource Typs
-          <a href="https://php.watch/versions/8.0/resource-CurlHandle">RFC</a>
+            Undurchsichtige Objekte anstelle von des resource Typs für
```
Comment (cmb69) [inline releases/8.0/de.php:430]:
```suggestion
            Opaque Objekte anstelle des resource Typs für
```
or even
```suggestion
            Objekte ohne Methoden anstelle des resource Typs für
```

File: releases/8.0/de.php [hunk 2]
```diff
@@ -424,11 +424,17 @@ public function __construct(
...
+            <a href="https://php.watch/versions/8.0/resource-CurlHandle">Curl</a>,
+            <a href="https://php.watch/versions/8.0/gdimage">Gd</a>,
+            <a href="https://php.watch/versions/8.0/sockets-sockets-addressinfo">Sockets</a>,
+            <a href="https://php.watch/versions/8.0/OpenSSL-resource">OpenSSL</a>,
+            <a href="https://php.watch/versions/8.0/xmlwriter-resource">XMLWriter</a>, und
+            <a href="https://php.watch/versions/8.0/xmlwriter-resource">XML</a>
+            extension
```
Comment (cmb69) [inline releases/8.0/de.php:437]:
```suggestion
            Extension
```

File: releases/8.0/pt_BR.php [hunk 3]
```diff
@@ -420,11 +420,17 @@ public function __construct(
                     </li>
                     <li>
                         Tipagem de funções internas
-                        <a href="https://externals.io/message/106522">RFC</a>
+                        <a href="https://externals.io/message/106522">discussão por email</a>
```
Comment (chapeupreto) [inline releases/8.0/pt_BR.php:423]:
```diff
- discussão por email
+ Discussão por email
```

**Comment by Girgias** [comment]
Merged as 9a8f7c6e54172644e284357bcaf0779e97f20664

**Pull request closed without merging**